### PR TITLE
Ansible configuration for the nav server

### PR DESCRIPTION
**Pull request by johanvdw**
Not sure where I should document this, but we require python 3.6 on the server. We are using artful on scaleway for this (actually a zesty upgraded to artful - as artful images are not offered).

Steps to import data after setup:
in virtual env (/opt/c3nav/env)
./manage.py migrate
./manage.py loaddata somefile.json
./manage.py collectstatic

### CODE REVIEW

File: ansible/playbooks/roles/maps-tileserver/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  file:
+    path: /opt/c3nav
+    state: directory
+    owner: root
+    group: root
+    mode: 0775
+
+- name: add c3nav user
+  user:
+    name: c3nav
```
Comment (SuperQ) [inline ansible/playbooks/roles/maps-tileserver/tasks/main.yml]: This should probably include:

```
system: yes
createhome: no
home: /opt/c3nav
```

File: ansible/playbooks/roles/maps-tileserver/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- name: install the c3nav-tiles uwsgi service
+  template:
+    src: c3nav-tiles.service
+    dest: /etc/systemd/system
+    owner: root
+    group: root
+    mode: 0644
+
+- name: enable c3nav-tiles service
+  service:
```
Comment (SuperQ) [inline ansible/playbooks/roles/maps-tileserver/tasks/main.yml]: I would recommend using `systemd` not `service` so you get `daemon_reload` correctly handled by default.

**Comment by SuperQ** [comment]
It might be worth using the `antoiner77.caddy` role instead of nginx.  It has a simpler config and better letsencrypt handling.

File: ansible/playbooks/roles/maps-tileserver/tasks/main.yml (hunk 1 above)
Comment (johanvdw) [reply ansible/playbooks/roles/maps-tileserver/tasks/main.yml]: done

**Comment by SuperQ** [comment]
Instead of creating the log file, why not make a `/var/log/c3nav/` directory and have things like `/var/log/c3nav/gunicorn.log`, `/var/log/c3nav/celery.log`.  This would tidy up the logging and make it less messy than having a `copy` invocation.

**Comment by johanvdw** [comment]
I exactly had the same idea

**Pull request merged**